docs/docs/features/soft-off.md

```diff
@@ -13,10 +13,15 @@ The power off is accomplished by putting the MCU into a "soft off" state. Power
 
 :::
 
-Once powered off, the keyboard will only wake up when:
+Causing a device to enter the soft-off state is done by:
 
-- You press the same button/sequence that you pressed to power off the keyboard, or
-- You press a reset button found on the keyboard.
+- Triggering a hardware-defined dedicated GPIO pin, if one exists;
+- Triggering the [Soft Off Behavior](../behaviors/soft-off.md) from the keymap.
+
+Once in the soft-off state, the device will only wake up when:
+
+- A hardware-defined dedicated GPIO pin is triggered, if one exists;
+- A reset button found on the device is pressed.
```

Comment:
This sounds alright to me, but to ensure technical correctness I'd appreciate if e.g. @petejohanson gave an OK.

Reply:
Removed LABEL= as requested. Mind you I have no idea what those do or why they were there, just following instructions. Agree on confirmation of correctness needed.

Reply:
Thanks, they used to be needed but they are no longer used as of #2028.

Reply:
Reading through this again on a different day, I have some unresolved questions on how it behaves and would like for them to be confirmed so that I can adjust the text to be clearer:
- Can you define multiple GPIO pins for soft-off? (Pretty sure yes)
- If multiple are defined, do you need to trigger the *same* soft-off pin to wake up, or can you trigger any? (Pretty sure any)
- Can a GPIO be defined for wakeup but not for activating soft-off (i.e. you could use &soft-off to put it to sleep, and press the index finger home to wakeup, I think this one is a no)

Reply:
> Reading through this again on a different day, I have some unresolved questions on how it behaves and would like for them to be confirmed so that I can adjust the text to be clearer:
> 
>     * Can you define multiple GPIO pins for soft-off? (Pretty sure yes)

I don't see why you couldn't... Not sure the value in doing so, but it should work.

> 
>     * If multiple are defined, do you need to trigger the _same_ soft-off pin to wake up, or can you trigger any? (Pretty sure any)

Any should do it... Under the hood, the only requirements for the *wake* is to generate any of the valid events that wake nRF52 from "System OFF": 
![image](https://github.com/zmkfirmware/zmk/assets/202695/aae2b602-80a9-4642-ba33-09c53b2fa085)

> 
>     * Can a GPIO be defined for wakeup but not for activating soft-off (i.e. you could use &soft-off to put it to sleep, and press the index finger home to wakeup, I think this one is a no)

I mean... in theory yes, if set up properly in the DTS. The two things are complementary, not strictly tied together.